src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -150,6 +150,18 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::PopStyleColor();
     }
 
+    // GPU
+    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+    if (ImGui::BeginTabItem("GPU")) {
+        ImGui::PopStyleColor();
+        ImGui::Checkbox("Hardware Flip", &host.cfg.hardware_flip);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Check the box to enable flipping from GPU side.\nPerformance and compabilities for commercial games increase.");
```

Comment:
Yeah, I think you meant "compatibility" instead of "compabilities." RPCS3's config might sound like this (if they had the option): "May increase performance. Required for some games."

Reply:
`"Enables texture flipping from GPU.\nMight Increase performance and is required for certain games."`

Reply:
Apparently it's not required. Just increases performance.

Reply:
hmm is requiert no, but only for commercail game, for hb need disable it, can causing flip image like i have see

Reply:
for exemple, stein:gate 0 without enable
![image](https://user-images.githubusercontent.com/5261759/63278991-8c8f3300-c2a8-11e9-9243-47276b9bd62e.png)

Reply:
So is really requiert for commercial game, but need disable for hombrew

Reply:
Sorry, I was missing a line. I will do a proper fix today. Just only increase performance

Reply:
no flip image for hombrew if enable ?

Reply:
Yeah for now compatibilities increase also  since somehombrews use premade shaders. Idk, increase it or not, you guys decide

Reply:
I forgot a line for steins gate 0 software to flip, will fix later

Reply:
but so is request or not for some commercial game ?

Reply:
Pushed change